README.md

```diff
@@ -61,10 +62,46 @@ Deploying the project is straightforward:
+
+1. **Install the required components**: Make sure you have Airflow and Prometheus ans Grafana installed and configured in your environment.
+
+2. **Mapping the configuration files**: Copy the contents of the `config_files/statsd/statsd.yaml` file from the monitoring project repository into your own Statsd configuration directory.
+
+3. **Modify Airflow configuration**: Open your Airflow configuration file (airflow.cfg) and add the following lines:
+
+   ```bash
+   AIRFLOW__METRICS__STATSD_ON: "True"
+   AIRFLOW__METRICS__STATSD_HOST: "statsd-exporter"
+   AIRFLOW__METRICS__STATSD_PORT: "9125"
+   AIRFLOW__METRICS__STATSD_PREFIX: "airflow"
+
+   Adjust the values based on your specific configuration.
+   ```
+
+4. **Configure Prometheus**: In your Prometheus configuration file (prometheus.yml), add the scrape configuration for the StatsD exporter:
+
+    ```bash
+    - job_name: 'statsd-exporter'
+      scrape_interval: 15s
+      static_configs:
+        - targets:
+          - 'statsd-exporter:9102'
+
+    Save the file after making the changes.
+    ```
+
+5. **Copy Grafana Dashboard JSON file**: From the `airflow-monitoring-and-alerting` repository, navigate to the config_files/grafana/var/lib/grafana/dashboards directory. Copy the JSON files of the desired dashboards you want to add to your Grafana instance.
+*Note: The specific directory path may vary depending on your Grafana setup.*
+
+By following these steps, you will integrate the monitoring stack from the `airflow-monitoring-and-alerting` repository into your existing Airflow and Prometheus setup. Make sure to adjust the configurations and paths according to your environment.
+
 ## Contributing 👥
 
-We welcome contributions to this repository! If you’re interested in contributing, please take a look at our [CONTIRIBUTION.md](https://github.com/data-burst/airflow_monitoring_and_alerting/blob/master/CONTRIBUTING.md) file for more information on how to get started. We look forward to collaborating with you!
+We welcome contributions to this repository! If you’re interested in contributing, please take a look at our [CONTIRIBUTION.md](https://github.com/data-burst/airflow-monitoring-and-alerting/blob/master/CONTRIBUTING.md) file for more information on how to get started. We look forward to collaborating with you!
 
 ## License 📄
 
-This repository is licensed under the MIT License, which is a permissive open-source license that allows for reuse and modification of the code with few restrictions. You can find the full text of the license in [this](https://github.com/data-burst/airflow_monitoring_and_alerting/license) file.
+This repository is licensed under the MIT License, which is a permissive open-source license that allows for reuse and modification of the code with few restrictions. You can find the full text of the license in [this](https://github.com/data-burst/airflow-monitoring-and-alerting/license) file.
```

Comment:
It appears that the link is corrupted. It is possible to link them relatively. (and also about CONTRIBUTING.md file)

Reply:
No, it's not corrupted. You are failed to see it, since it is not merged to the main branch yet!